ui/pages/confirmations/components/confirm/info/personal-sign/siwe-sign/siwe-sign.tsx

```diff
@@ -44,7 +44,7 @@ const SIWESignInfo: React.FC = () => {
   return (
     <>
       <ConfirmInfoRow label={t('message')}>
-        <ConfirmInfoRowText text={statement} />
+        <ConfirmInfoRowText text={statement || ''} />
```

Comment:
```suggestion
        <ConfirmInfoRowText text={statement ?? ''} />
```

Reply:
oh nice! I can update this in a subsequent alert PR to not need re-reviews